app/templates/view_supplier_services.html

```diff
@@ -25,39 +25,53 @@
   {% with heading = "Services" %}
     {% include "toolkit/page-heading.html" %}
   {% endwith %}
-  {% call(item) summary.list_table(
-    services,
-    caption="Services",
-    empty_message="This supplier has no services on the Digital Marketplace",
-    field_headings=[
-      'Name',
-      'Service ID',
-      'Framework',
-      'Lot',
-      'Status',
-      summary.hidden_field_heading("Action")
-    ],
-    field_headings_visible=True
-  ) %}
-    {% call summary.row() %}
-      {{ summary.service_link(item.serviceName, '/' + item.frameworkFramework + '/services/' + item.id|string) }}
-      {{ summary.text(item.id) }}
-      {{ summary.text(item.frameworkName) }}
-      {{ summary.text(item.lotName) }}
-      {% call summary.field() %}
-        <span class="service-status-{{ item.status }}">
-          {% if item.status == "published" %}
-            Public
-          {% elif item.status == "enabled" %}
-            Private
-          {% elif item.status == "disabled" %}
-            Removed
-          {% else %}
-            {{ item.status|title }}
-          {% endif %}
-        </span>
+  {% set framework_services_grouper = dict(services | groupby('frameworkSlug')) %}
+  {% if not framework_services_grouper %}
+    {% call(item) summary.list_table(
+      [],
+      caption="Services",
+      empty_message="This supplier has no services on the Digital Marketplace.",
+      field_headings=['Name', 'ID', 'Lot', 'Status', summary.hidden_field_heading("Action")],
+    field_headings_visible=True)
+    %}{% endcall %}
+  {% endif %}
+  {% for framework in frameworks | sort(attribute='id', reverse=True) %}
```

Comment:
There's a lot of grunt-work done in the template, but it's probably fine.

Reply:
Done, moved to view